src/app/public/modules/text-editor/toolbar/text-editor-toolbar.component.ts

```diff
@@ -91,10 +95,29 @@ export class SkyTextEditorToolbarComponent implements OnInit {
   public set styleState(value: SkyTextEditorStyleState) {
     this._styleState = value;
     if (value.font !== this.styleStateFontName) {
-      this.styleStateFontName = this.getFontName(value.font);
+      if (value.font === '\"Blackbaud Sans\", Arial, sans-serif') {
+        this.styleStateFontName = this.getFontName('Blackbaud Sans');
+      } else {
+        this.styleStateFontName = this.getFontName(value.font);
+      }
+    }
+  }
+
+  @Input()
+  public set disabled(value: boolean) {
+    const coercedValue = SkyFormsUtility.coerceBooleanProperty(value);
+    if (coercedValue !== this.disabled) {
+      this._disabled = coercedValue;
+      this.changeDetector.markForCheck();
     }
   }
 
+  public get disabled() {
+    return this._disabled;
+  }
+
+  private _disabled: boolean = false;
```

Comment:
`private` properties that are accessed via getters/setters should be ordered under all other private properties. See https://host.nxt.blackbaud.com/skyux-team-docs/style#class-properties

Reply:
fixed in "[correct order of '_' private properties](https://github.com/blackbaud/skyux-text-editor/pull/21/commits/a94669b5faf7179e461d200e9953a167752a695f)"